### PR TITLE
[Axonius 1.0.5] Additional Axonius Integration Commands

### PR DESCRIPTION
**Pull request by bryce-ax**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: N/A

## Description
Added 4 new commands to existing Axonius integration:
- `!axonius-get-tag` to add tag to devices/users
- `!axonius-remove-tag` to remove tag from devices/users
- `!axonius-get-tags` to return all devices/users tags
- `!axonius-get-savedqueries` to return all devices/users saved queries

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by bziser — COMMENTED** [review]
Thank you for the contribution! Looks good.
Please add unit tests for the new commands.

**Comment by bryce-ax** [comment]
Roger that. We will add additional test cases early next week.

**Comment by kgal-pan** [comment]
> Roger that. We will add additional test cases early next week.

@bryce-ax - Any updates on this?

**Comment by bryce-ax** [comment]
The Axonius client that we use in our integration is not very conducive to mock data. We are working on the best solution and should have an update by EOD.

File: Packs/Axonius/Integrations/Axonius/TestData/Expected_data.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+EXPECTED_USERS_SQS = [
```
Comment (kgal-pan) [inline Packs/Axonius/Integrations/Axonius/TestData/Expected_data.py:1]: Run the following to add the secret to ignore file.

```bash
echo "dummy@axonius.com" > Packs/Axonius/.secrets-ignore
```

Conversely, remove the email completely.

**Comment by kgal-pan** [comment]
[Update upstream with `master`](https://app.circleci.com/pipelines/github/demisto/content/155267/workflows/212fdc2f-8ccd-40ad-af90-0a50473f491c/jobs/502463?invite=true#step-109-9):

```bash
/Utils/git_pull_master_into_fork.sh
```

File: Packs/Axonius/Integrations/Axonius/TestData/Expected_data.py (hunk 1 above)
Comment (bryce-ax) [reply Packs/Axonius/Integrations/Axonius/TestData/Expected_data.py:1]: done

File: Packs/Axonius/Integrations/Axonius/Axonius_test.py [hunk 2]
```diff
@@ -1,9 +1,97 @@
 """Axonius Integration for Cortex XSOAR - Unit Tests file."""
 import inspect
+import pytest
+from unittest import TestCase
 
 import axonius_api_client as axonapi
+from axonius_api_client.connect import Connect
+from axonius_api_client.api import Devices
+from axonius_api_client.api import Users
```
Comment (kgal-pan) [inline Packs/Axonius/Integrations/Axonius/Axonius_test.py:9]:
```suggestion
import axonius_api_client as axonapi
```

**Comment by kgal-pan** [comment]
Missing coverage in the [UTs](https://app.circleci.com/pipelines/github/demisto/content/155267/workflows/212fdc2f-8ccd-40ad-af90-0a50473f491c/jobs/502464?invite=true#step-108-20):

```
[ERROR] file: Packs/Axonius/Integrations/Axonius/Axonius.py unittests coverage should reach at least 70.0 currently 36.31.
```

Feel free to use [the coverage report](https://output.circle-artifacts.com/output/job/d71e9046-7810-4886-88b6-bd0a6c8012aa/artifacts/0/artifacts/coverage_report/html/d_b2d1eba43e507bc1_Axonius_py.html) to see which areas are missing UT code coverage.

File: Packs/Axonius/Integrations/Axonius/Axonius_test.py (hunk 2 above)
Comment (kgal-pan) [reply Packs/Axonius/Integrations/Axonius/Axonius_test.py]: [`e35ebf7` (#18373)](https://github.com/demisto/content/pull/18373/commits/e35ebf75faea8140b1ea3e39a451fa2095089b85)

**Review by bziser — APPROVED** [review]
Great job! Thanks for the contribution

**Pull request merged**